test1Tile.py

```diff
@@ -0,0 +1,212 @@
+            psfdict= {}
+
+            for s in ['g','r','i','z']:
+                sw=s+"w"
+                sp=s+"p"
+                a=90
+
+                ext=objext[s]
+                header = fits[ext].read_header()
+                imdict[s] = fits[ext].read()[a:-a,a:-a]
+
+                ext=objext[sw]
+                header = fits[ext].read_header()
+                sigdict[s] = (fits[ext].read()[a:-a,a:-a])**-2
+
+                ext=objext[sp]
+                header = fits[ext].read_header()
+                psfdict[s] = fits[ext].read()
+            print objectN
+            #["3017249430"]
+
+            #BR=BlueRings(imdict,sigdict,psfdict)
+            
+            if objectN in interesting or int(objectN)>3017265667:
+                BR=BlueRings(imdict,sigdict,psfdict)
+                inter=BR.plot()
+                if inter!="":interesting.append(objectN)
+                print "[",
+                for inter in interesting:
+                    print "\"%s\","%inter,
+                print"]"
+
+
+            """
+            plt.subplot(131)
+            color = colorImage.ColorImage()
+            colorimage = color.createModel(imdict['g'],imdict['r'],imdict['i'])
+            plt.imshow(colorimage,interpolation="none")
+            plt.subplot(132)
+
+            #psfmode="crossconvolve"
+            #RF=RingFinder(imdict['g'],sigdict['i']**-2,sigdict['g']**-2,imdict['i'],psfdict['g'],psfdict['i'],0.265,1e12,1e12,visualize=False,psfmode=psfmode)
+            #plt.imshow(RF.Dshow,interpolation="none")
+            #plt.show()
+
+            BR=BlueRings(imdict,sigdict,psfdict)
+
+            colorimage = color.colorize(BR.subdict['g'],BR.subdict['r'],BR.subdict['i'])
+            plt.imshow(colorimage,interpolation="none")
+
+            plt.subplot(133)
+            BR.subdict['g'][BR.subdict['g']<0]=0
+            plt.imshow(BR.subdict['g'],interpolation="none")
+
+
+            plt.draw()
+            raw_input()
+            """
+
+if __name__ == "__main__":
+    dir_input = "/home/ttemp/BlueRings/data/"
+    file_input = "DES0005-0041_cutouts.fits"
+    fitsfile = dir_input+file_input
+
+    tile=RingFindforTile()
+    tile.produce(fitsfile)
+exit() 
+
+
+a=40
+for j in range(100):
+    imgg=hdulist[0+j*15].data[a:-a,a:-a]
+    sigg=hdulist[1+j*15].data[a:-a,a:-a]
+    psfg=hdulist[2+j*15].data
+    imgr=hdulist[3+j*15].data[a:-a,a:-a]
+    sigr=hdulist[4+j*15].data[a:-a,a:-a]
+    psfr=hdulist[5+j*15].data
+    imgi=hdulist[6+j*15].data[a:-a,a:-a]
+    sigi=hdulist[7+j*15].data[a:-a,a:-a]
+    psfi=hdulist[8+j*15].data
+    imgz=hdulist[9+j*15].data[a:-a,a:-a]
+    sigz=hdulist[10+j*15].data[a:-a,a:-a]
+    psfz=hdulist[11+j*15].data
+
+    color = colorImage.ColorImage()
+    colorimage = color.createModel(imgg,imgr,imgi)
+  #plt.imshow(colorimage,interpolation="none")
+  #plt.show()
+    psfmode="none"
+    RF=RingFinder(imgg,imgi,sigg,sigi,psfg,psfi,0.265,1e12,1e12,visualize=False,psfmode=psfmode)
```

Comment:
These `RingFinder` calls could be placed inside a method that took a `cutout` object, and returned a corresponding `cutout` object with the red light removed. That subtracted cutout would then know how to make a composite of itself...